src/validation-error-message-formatter.js

```diff
@@ -285,13 +285,6 @@ exports.supportedContentTypesAttachmentReferenceViolation = function(itemPath, e
   return 'attachment reference "' + itemPath + '" must have a supported content type (' + contentTypesString + ')';
 };
 
-/**
- * DEPRECATED. Use supportedContentTypesAttachmentReferenceViolation instead.
- */
-exports.supportedContentTypesAttachmentViolation = function(itemPath, expectedContentTypes) {
```

Comment:
Since the validation-error-message-formatter module was moved to this location and the package has not been released since then, there is no good reason to keep this member around.